hoomd/HOOMDVersion.cc

```diff
@@ -173,4 +173,15 @@ std::string BuildInfo::getInstallDir()
     return std::string(HOOMD_INSTALL_PREFIX) + "/" + std::string(PYTHON_SITE_INSTALL_DIR);
     }
 
+std::pair<unsigned int, unsigned int> BuildInfo::getFloatingPointPrecision()
+    {
+#if defined(SINGLE_PRECISION)
+    return std::make_pair(32, 32);
+#elif !defined(SINGLE_PRECISION) && defined(ENABLE_HPMC_MIXED_PRECISION)
+    return std::make_pair(64, 32);
+#elif !defined(SINGLE_PRECISION) && !defined(ENABLE_HPMC_MIXED_PRECISION)
+    return std::make_pair(64, 64);
```

Comment:
Some of the conditions look redundant. Is that for being explicit?

```suggestion
#elif defined(ENABLE_HPMC_MIXED_PRECISION)
    return std::make_pair(64, 32);
#elif
    return std::make_pair(64, 64);
```

Reply:
I'm planning ahead. Per request by users in #355, all 4 options will possible when we enable MD mixed precision.